cephprimarystorage/ansible/cephp.py

```
@@ -166,18 +169,20 @@
 
-# name: make sure virtualenv has been setup
-command = "[ -f %s/bin/python ] || virtualenv-2.7 --system-site-packages %s " % (virtenv_path, virtenv_path)
-run_remote_command(command, host_post_info)
+if not py_version:
+    # name: make sure virtualenv has been setup
+    command = "python3.11 -m venv %s --system-site-packages" % virtenv_path
+    run_remote_command(command, host_post_info)
 
 # name: install python pkg and replace ceph python path
-replace_content(ceph_file_path, "regexp='/usr/bin/env python' replace='/usr/bin/python2.7'", host_post_info)
+replace_content(ceph_file_path, "regexp='/usr/bin/env python' replace='/usr/bin/python3.11'", host_post_info)
```

Ruff (0.15.2)

[error] `replace_content` may be undefined, or defined from star imports

(F405)